src/cc/CMakeLists.txt

```diff
@@ -47,10 +47,15 @@ add_library(bcc-shared SHARED
 set_target_properties(bcc-shared PROPERTIES VERSION ${REVISION_LAST} SOVERSION 0)
 set_target_properties(bcc-shared PROPERTIES OUTPUT_NAME bcc)
 
-add_library(bcc-loader-static STATIC ${bcc_sym_sources} ${bcc_util_sources})
+if(ENABLE_USDT)
+  set(bcc_usdt_sources usdt/usdt.cc usdt/usdt_args.cc)
+  # else undefined
+endif()
+
+add_library(bcc-loader-static STATIC ${bcc_sym_sources} ${bcc_util_sources} ${bcc_usdt_sources})
```

Comment:
bcc_usdt_sources has been added below directly to bcc-static and it should work. So the bcc_usdt_sources at bcc-loader-static can be dropped. Could you give a try?

Reply:
You're right, thanks, I've made the change and it works. Pushed.